File: homework 3/6/app/src/main/java/com/example/classwork04/MainActivity.java
package com.example.classwork04;

import androidx.appcompat.app.AppCompatActivity;
import android.os.Bundle;

import com.example.classwork04.continent.ContinentFragment;
import com.example.classwork04.databinding.ActivityMainBinding; // Import the generated binding class

public class MainActivity extends AppCompatActivity {
    private ActivityMainBinding binding;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityMainBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
        getSupportFragmentManager().beginTransaction().add(R.id.container, new ContinentFragment()).commit();
    }
}

File: homework 3/9/app/src/main/java/com/example/hw_03_07/MainActivity.kt
package com.example.hw_03_07

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        if (savedInstanceState == null){
            supportFragmentManager.beginTransaction().add(R.id.container1, FirstFragment()).commit()
            supportFragmentManager.beginTransaction().add(R.id.container2, SecondFragment()).commit()
        }

    }
}